src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/types';
+
+const initialTodos: Todo[] = todosFromServer;
+
+function getNewTodoId(todos: Todo[]) {
+  const maxId = Math.max(...todos.map(todo => todo.id));
```

Comment:
Ensure that `Math.max` handles the case where `todos` might be empty. You can use `Math.max(0, ...todos.map(todo => todo.id))` to start `id` from 1 if the list is empty.